inst/extdata/default.yaml

```diff
@@ -1,5 +1,5 @@
-xaxs: "i"
-yaxs: "i"
+xaxs: "r"
```

Comment:
I was going to say that and changed my mind. Everyone should know though that all of our examples will now look different. I think we leave it as "r" and bump up the priority to fix the user-specified limit stuff.